src/styles/header.scss

```diff
@@ -0,0 +1,36 @@
+@import "utils.scss";
+
+.header__container {
+    height: 10rem;
+    @media (min-width: $breakpoint-large){
+        height: 25rem;
+    }
+}
+
+.stripe__container {
+
+  @media (min-width: $breakpoint-large) {
```

Comment:
kindly use the predefined mixins for large screen in  utils.scss by applying @include breakPoint(large){}